modules/nw-control-dns-records-public-hosted-zone-infoblox.adoc

```diff
@@ -53,20 +53,21 @@ spec:
         name: infoblox-credentials
       gridHost: ${INFOBLOX_GRID_PUBLIC_IP}
       wapiPort: 443
-      wapiVersion: "2.3.1"
+      wapiVersion: "2.3.1" <3>
```

Comment:
As discussed, a concrete version is better to be replaced with a placeholder (`<version>`) to let the user decide which version suits better their installation.